src/background.js

```diff
@@ -196,8 +226,27 @@ const rollout = {
     return data[name];
   },
 
-  async setSetting(name, value) {
+  async setSetting(name, value, makeDurable) {
     await browser.storage.local.set({[name]: value});
```

Comment:
Why still write to local storage at all? What if local storage and prefs get out of sync? Which one do we read? Which one wins?

Reply:
This is the larger question that applies to both comments above. I'm opting for pref-setting only going forward. Durable prefs, no local storage reset issues, and better code breadcrumbs for the future. I'll still add a JSDOC comment!

Reply:
I have the same question as @mythmon .

Aside, this is inconsistent with `getSetting()`. If we do this here, then in `getSetting()` we should try reading from local storage first, then if that fails, try the durable pref.

Reply:
@nhi-nguyen @mythmon Rather than depend on something that could end up being inconsistent, I recommend only using prefs. (Which is what this PR now does)

PR #175 addresses moving all found local storage items to prefs.